cmd/create.go

```diff
@@ -194,8 +195,8 @@ func (cc *createCmd) detectLanguage() (*config.DraftConfig, string, error) {
 				if lang.Language == "Java" {
 
 					selection := &promptui.Select{
-						Label: "Linguist detected Java, are you using maven or gradle?",
-						Items: []string{"gradle", "maven"},
+						Label: "Linguist detected Java, are you using maven or gradle or gradle wrapper?",
+						Items: []string{"gradle", "maven", "gradlew"},
```

Comment:
Nice, thanks for this! I wonder how it was happening before, the difference lies in the fact that ./gradlew indicates you are using a gradle wrapper. The wrapper is generally part of a project and it facilitates installation of gradle. so was it something working with manual cradle installation.